14/umbraco-cms/fundamentals/backoffice/property-editors/built-in-umbraco-property-editors/content-picker.md

```diff
@@ -1,41 +1,195 @@
 # Content Picker
 
-`Alias: Umbraco.ContentPicker`
+`Alias: Umbraco.MultiNodeTreePicker`
 
-`Returns: IPublishedContent`
+`Returns: IEnumerable<IPublishedContent>`
 
-The content picker opens a panel to pick a specific page from the content structure. The value saved is the selected nodes [UDI](../../../../reference/querying/udi-identifiers.md)
+## Settings
+
+The Content Picker allows you to configure the type of tree to render and what part of the tree that should be rendered. For content it allows you to select a dynamic root node based on the current document using the content picker.
+
+### Node type
+
+Set the type of node, the root node of the tree, or query for the root node.
+
+For querying content you can specify a dynamic root:
+
+<figure><img src="../../images/mntp_node_type.png" alt=""><figcaption></figcaption></figure>
+
+When you specify the dynamic root, you are able to navigate the tree relative to a node.
+
+<figure><img src="../../images/mntp_node_type_dynamic_root.png" alt=""><figcaption></figcaption></figure>
+
+First you have to specify a origin, from where the query will start.
+
+![Content Picker Data Type Definition](../../images/mntp\_node\_type\_dynamic\_root\_origin.png)
+
+You have the following options:
+
+* Root
+  * The root is the first level item of the current nodes subtree.
+* Parent
+  * The parent is the nearest ancestor of the current node.
+* Current
+  * The current node. Be aware a picker that uses the current node, cannot pick anything when the current node is created, because it do not have any children.
+* Site
+  * The nearest ancestor of the current node that has a domain assigned.
+* Specific node
+  * A specific node that you have to choose in the tree
```

Comment:
```suggestion
* Root - The root is the first level item of the subtree of the current node.
* Parent - The parent is the nearest ancestor of the current node.
* Current - The current node. Be aware a picker that uses the current node, cannot pick anything when the current node is created, because it does not have any children.
* Site - The nearest ancestor of the current node that has a domain assigned.
* Specific node - A specific node that you have to choose in the tree
```